### ISSUE TITLE
how to built code?

### ISSUE THREAD

**Issue by TrumanDu** [lmenezes/cerebro#194]
**i just down master code,build on sbt 1.0.0,but find some issue**:
```
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  ::          UNRESOLVED DEPENDENCIES         ::
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  :: com.typesafe.play#sbt-plugin;2.5.10: not found
[warn]  :: com.typesafe.sbt#sbt-rjs;1.0.4: not found
[warn]  :: com.typesafe.sbt#sbt-digest;1.0.0: not found
[warn]  :: com.typesafe.sbt#sbt-gzip;1.0.0: not found
[warn]  :: com.typesafe.sbt#sbt-native-packager;1.2.0-M9: not found
[warn]  :: com.eed3si9n#sbt-buildinfo;0.6.1: not found
```
```
https://repo.typesafe.com/typesafe/releases/
```
this repo not availables

pls help me,how to built code? how to develop?
thanks

**Comment by lmenezes** [comment]
@TrumanDu hum, I have no idea why you get this. I have just downloaded sbt 1.0(was previously running on 13.X), delete my whole ivy cache, and everything worked out just fine. Perhaps some firewall or network issues on your side?

**Comment by lmenezes** [comment]
@TrumanDu ok, I actually managed to get your error by setting the **sbt.version** on **build.properties** to 1.0(rather than 0.13.11). Is that what happened to you?

**Comment by TrumanDu** [comment]
Yeah,you have got it. sbt version now is 1.0,0.13.11version unused.have some suggest for me?thanks

**Comment by lmenezes** [comment]
It seems that some of the plugins aren't yet available for sbt 1.0, so just use the 0.13.11 and all should work fine.
